entries/royalholloway.ac.uk.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Royal Holloway, University of London": {
+    "domain": "www.royalholloway.ac.uk",
+    "url": "https://www.royalholloway.ac.uk",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```

Reply:
Ah copied this from another json file. Already was confused about the duplication.